docs/source/topics/ref_minimum-system-requirements.adoc

```diff
@@ -9,13 +9,14 @@
 {prod} requires the following system resources:
 
 * 4 virtual CPUs (vCPUs)
-* 8 GB of memory
+* 9 GB of free memory
```

Comment:
👍

Reply:
I was actually mulling about this sentence... especially with the note that follows.